ci-operator/step-registry/ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
I think that you added this in the wrong place, you should add/edit these files following the path:
- step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

Reply:
Actually, this is duplicated, since you also added them in the right path

Reply:
Right I'm removing the two dups : 
D  ci-operator/step-registry/ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh
D  ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

and leaving only the right one : 
ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh